app/views/layouts/application.html.erb

```diff
@@ -14,6 +14,8 @@
     <%= vite_javascript_tag "packs/application", media: 'all', "data-turbo-track": "reload" %>
     <%= vite_stylesheet_tag "packs/application", media: 'all', "data-turbo-track": "reload" unless ViteRuby.instance.dev_server_running? %>
 
+    <%= display_meta_tags title: current_page.data["title"] %>
```

Comment:
Browser show the first `<title`> they encounter in the `<head`>. It looks like we currently have two. Can we remove the existing `<title>` tag on L4 so it shows the proper one 🙌🏼